Trust/Trust/Main/Home/HomeView.swift

```diff
@@ -8,11 +8,34 @@
 import SwiftUI
 
 struct HomeView: View {
+    // MARK: - Public Properties
+
+    var navigator: Navigator
+
+    // MARK: - Body
+
     var body: some View {
-        Text(verbatim: "Home")
+        VStack {
+            Text(verbatim: "Home")
+        }
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
```

Comment:
In the official Trust Wallet app, the settings button is placed on the top-left of the navigation bar but presents the settings view from the right to the left. 

This is an unusual behavior for an iOS app, so I took the liberty of moving it to the top-right corner instead. But please let me know if there was a specific reason for the placement. :)